eZ Platform/Admin UI/Field Type Edit/Requirements.md

```diff
@@ -0,0 +1,35 @@
+# Content Editing
+
+## Objective
+
+The objective (which may be supported by prototypes or examples), is to identify and decide on the best approach to implement CRUD (create, read, update and delete) operations on Field Types within the user interface(s) and APIs exposed by eZ Platform in eZ Platform v2.x.
+
+The underlying goals are:
+
+- Simplify how to create a new Field Type (wether it is for eZ Systems' own needs when creating Field Types in the core platform, or developers customizing the platform by creating custom Field Types)
+- Reduce risk of errors and simplify maintenance by limiting the number of places where UI behavior is defined
+- Increase speed for developers to develop and maintain Field Types with a similar efficiency level as what was offered by eZ Publish Legacy, or better.
```

Comment:
André, no offense but I have to disagree. The value eZ Platform delivers to existing eZ partners and developers is directly tied to how what can be done with eZ Platform benchmark s with the way things were eZ Publish legacy. There is a replacement project base, and there are also new projects by skilled eZ Publish developers that are directly impacted by that. In both cases, the question is "should I go with eZ Platform?". Feedback from legacy developers was that field types, even if not as easy as it could be and not perfect, was delivering value, and to speak frankly, doing the trick. That's why I think the requirement should at the very least be "as simple". I think we should keep it as an expected outcome of 2.x. But in any case, it looks like we aim for much better now, so we should be good :-)